app/javascript/pages/country/data/weeks.json

```diff
@@ -1,4 +1,8 @@
 [
+  {
+    "label": "4 weeks",
+    "value": 4
```

Comment:
this affects the glad-alerts widget. Is this right?

Reply:
Ah cheese 🤔 . Yes I should have put in the .env. And yes weeks selector needs updating. 🏎 💨

Reply:
Fixed!